core/src/test/java/io/questdb/test/cairo/mv/MatViewTest.java

```diff
@@ -2603,61 +2603,118 @@ public void testEnableDedupWithSameKeysDoesNotInvalidateMatViews() throws Except
     }
 
     @Test
-    public void testEstimateRowsPerBucket() {
+    public void testEstimateBucketsForRows() {
+        final long targetRows = 100_000L;
```

Comment:
How about changing this to 1M which is the current default for rowsPerQuery? This way we may have less `(minimum)` results.